homework-submission/week1/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './App.css';
+import ToDoList from './ToDoList';
+
+class App extends Component {
+  render() {
+    return (
+      <div className='App-header'>
+        <ToDoList></ToDoList>
```

Comment:
if there in no text or data inside the tag then you don't need the close tag
i meant you can use
<  ToDoList />  
instead of < ToDoList >< / ToDoList >